composer.json

```diff
@@ -2,7 +2,18 @@
     "name": "cakephp/cakephp",
     "description": "The CakePHP framework",
     "type": "library",
-    "keywords": ["framework"],
+    "keywords": [
+        "framework",
+        "mvc",
+        "rapid-development",
+        "conventions over configuration",
+        "dry",
+        "orm",
+        "form",
+        "validation",
+        "rest-api",
```

Comment:
They don't provide a rest-api. you can build one with it (using the plugin that actually has those keywords listed), but so can you build a "youtube reloaded", a "math forum for geeks" or a "dating portal" :) None of those are candidates for keywords, though.